server/config.mjs

```diff
@@ -0,0 +1,13 @@
+export default class Config {
```

Comment:
Any reason why you're using a class and not a simple object? It's probably more future proof..

Reply:
I was thinking there may be some config logic that requires calling functions. It's probably easier to add some to a class.